app/javascript/advertisements/index.js

```diff
@@ -0,0 +1,62 @@
+import { setUpliftUrls, detectUplift } from './uplift'
+import {
+  closeCodefund,
+  codefundClosed,
+  dispatchCodefundEvent,
+  elementVisible
+} from './helpers'
+
+const showCodeFundAdvertisement = (
```

Comment:
In some places you use `Codefund` and in others `CodeFund`. Any reason?

Reply:
Mostly artifacts from previous code. Will get everything consistent before this wraps up.